app/src/main/java/xyz/jhughes/laundry/fragments/MachineFragment.java

```diff
@@ -164,10 +163,15 @@ public void onResponse(Call<ArrayList<Machine>> call, Response<ArrayList<Machine
                 @Override
                 public void onFailure(Call<ArrayList<Machine>> call, Throwable t) {
                     //likely a timeout -- network is available due to prev. check
-                    showErrorDialog(getString(R.string.error_server_message));
-                    mSwipeRefreshLayout.setRefreshing(false);
-                    isRefreshing = false;
-                    alertNetworkError();
+                    if (isAdded() && getActivity() != null) {
+                        showErrorDialog(getString(R.string.error_server_message));
+
+                        mSwipeRefreshLayout.setRefreshing(false);
+                        isRefreshing = false;
+                        alertNetworkError();
+                    } else {
+                        AnalyticsHelper.sendEventHit(AnalyticsHelper.ERROR, "onFailure", "MachineFragment's Activity Null");
```

Comment:
There is a SendErrorHit that might be better here. We can create an exception here but not throw it. That should create and send a stack trace to GA for us. 

Also you can set it to not fatal so we can still ignore them in GA when necessary.